operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```
@@ -157,13 +157,33 @@ local utils = import 'mixin-utils/utils.libsonnet';
     },
 
   // hiddenLegendQueryPanel is a standard query panel designed to handle a large number of series.  it hides the legend, doesn't fill the series and
-  //  sorts the tooltip descending
-  hiddenLegendQueryPanel(queries, legends, legendLink=null)::
+  // shows all values on tooltip, descending.  Also turns on exemplars, unless 4th parameter is false.
```

```suggestion
  // shows all values on tooltip, descending. Also turns on exemplars, unless 4th parameter is false.
```

Can you also mention that it only work with the timeseries panels, not with the legacy graph panels?